scc/cert/committee_cert_json.go

```diff
@@ -0,0 +1,52 @@
+package cert
+
+import (
+	"fmt"
+
+	"github.com/0xsoniclabs/sonic/scc"
+	"github.com/0xsoniclabs/sonic/scc/bls"
+	"github.com/0xsoniclabs/sonic/utils/jsonhex"
+)
+
+type committeeCertificateJson struct {
+	ChainId   uint64          `json:"chainId"`
+	Period    uint64          `json:"period"`
+	Members   []scc.Member    `json:"members"`
+	Signers   jsonhex.Bytes   `json:"signers"`
+	Signature jsonhex.Bytes96 `json:"signature"`
+}
+
+func (c committeeCertificateJson) String() string {
```

Comment:
nit: I still think this is unnecessary for an internal type and `%v` or `%+v` based on reflection does probably a similar job;